modules/Sources/Features/Home/HomeStore.swift

```diff
@@ -206,23 +206,18 @@ public struct HomeReducer: ReducerProtocol {
             case .foundTransactions:
                 return .fireAndForget { reviewRequest.foundTransactions() }
 
-            case .updateDestination(.profile):
-                state.profileState.destination = nil
-                state.destination = .profile
-                return .none
-                
             case .updateDestination(let destination):
                 state.destination = destination
                 return .none
                 
-            case .profile(.back):
-                state.destination = nil
-                return .none
+//            case .addressDetails(.back):
+//                state.destination = nil
+//                return .none
```

Comment:
Should we remove these lines rather than comment out?

Reply:
Removed